src/app/parcels/add/formSections/CollectionDateCard.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { CardProps, onChangeDateOrTime, errorText } from "@/components/Form/formFunctions";
+import GenericFormCard from "@/components/Form/GenericFormCard";
+import StyledDatePicker from "@/components/DataInput/DatePicker";
+import { ErrorText } from "@/components/Form/formStyling";
+
+const CollectionDateCard: React.FC<CardProps> = ({
+    fieldSetter,
+    errorSetter,
+    formErrors,
+    fields,
+}) => {
+    return (
+        <GenericFormCard
+            title="Collection Date"
+            required={true}
+            text="What date is the client collecting their parcel?"
+        >
+            <>
+                <StyledDatePicker
+                    onChange={(value: any): void => {
```

Comment:
Does this have to be type `any`? I think using `unknown` might be better in terms of type safety? (not sure)